### PR TITLE
Fix fedora-atomic after fedora-28 refresh

### PR DESCRIPTION
**Pull request by martinpitt**
See individual commits

 * [x] image-refresh fedora-atomic

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh in progress on cockpit-task-centos-ci-2.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-9969-20180903-111515/

**Comment by cockpituous** [comment]
image-refresh fedora-atomic done: https://github.com/cockpituous/cockpit/commits/image-refresh-fedora-atomic-20180903-112525

**Comment by martinpitt** [comment]
Yay, this finally seems to work..

File: bots/images/scripts/lib/atomic.setup [hunk 1]
```diff
@@ -44,7 +44,7 @@ if [ "$ID" = "rhel" ]; then
     REPO="rhel-7-server-extras-rpms"
     ID="rhel7"
 fi
-docker run --rm --volume=/etc/yum.repos.d:/etc/yum.repos.d:z --volume=/root/rpms:/tmp/rpms:rw,z "$ID:$VERSION_ID" /bin/sh -cex "yum install -y findutils createrepo && yum install -y --downloadonly --enablerepo=$REPO libssh && find /var -name '*.rpm' | while read rpm; do mv -v \$rpm /tmp/rpms; done; createrepo /tmp/rpms"
+docker run --rm --volume=/etc/yum.repos.d:/etc/yum.repos.d:z --volume=/root/rpms:/tmp/rpms:rw,z "$ID:$VERSION_ID" /bin/sh -cex "yum install -y findutils createrepo yum-utils && (cd /tmp/; yumdownloader --enablerepo=updates libssh) && find /tmp -name '*.$(uname -m).*rpm' | while read rpm; do mv -v \$rpm /tmp/rpms; done; createrepo /tmp/rpms"
```
Comment (martinpitt) [inline bots/images/scripts/lib/atomic.setup]: Whoops, this should use `$REPO`, not `"updates"`.

**Comment by martinpitt** [comment]
Ignoring the ovirt flake, in the previous test run it worked.

**Pull request merged**